functions/Restore-DbaDatabase.ps1

```diff
@@ -483,7 +538,7 @@ function Restore-DbaDatabase {
 			Write-Message -Level Verbose -Message "Starting FileSet"
 			if (($FilteredFiles.DatabaseName | Group-Object | Measure-Object).count -gt 1) {
 				$dbs = ($FilteredFiles | Select-Object -Property DatabaseName) -join (',')
-				Write-Warning "$FunctionName - We can only handle 1 Database at a time - $dbs"
+				Stop-Function -silent:$silent -message "We can only handle 1 Database at a time - $dbs"
```

Comment:
Silent not needed - unless your command does not have a $Silent primary param. All external/exported commands must.